### PR TITLE
Adding OurTeam profile for Chris Schillinger

### PR DESCRIPTION
**Pull request by ChrisSchillingerTheLadders**

### CODE REVIEW

File: source/_includes/ourteam/chrisschillinger.markdown [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<div class="profile-container">
+  <div class="profile-thumb">
+    {% gravatar cschillinger@theladders.com %}
+  </div>
+  <div class="profile-content">
+    <strong>Chris Schillinger</strong> is a Sr. Software Engineer at TheLadders who loves working on systems-level code in any language. He spent most of his career programming backend services for online videogames before deciding to switch to development for the World Wide Web. When not programming, he might be found; exploring tasty beer and food establishments, swing dancing, traveling the world, and/or playing D&D.
```
Comment (zsiec) [inline source/_includes/ourteam/chrisschillinger.markdown]: Swing dancing! :+1:

Comment (ChrisSchillingerTheLadders) [inline source/_includes/ourteam/chrisschillinger.markdown]: It's super fun!

Comment (zsiec) [inline source/_includes/ourteam/chrisschillinger.markdown]: Agreed! It's nice to have another Lindy Hopper at TheLadders. :smile:

Comment (jeremyladders) [inline source/_includes/ourteam/chrisschillinger.markdown]: The most important question: What edition of D&D?

**Pull request merged**